services/enrichment/manager/webpack.config.js

```diff
@@ -4,19 +4,28 @@ const path = require('path');
 module.exports = {
   entry: slsw.lib.entries,
   target: 'node',
+  mode: slsw.lib.webpack.isLocal ? 'development' : 'production',
+  optimization: {
+    minimize: false,
```

Comment:
In prod, you'll probably want to optimize the code

Reply:
Honestly, I did initially. The minimization didn't really result in gain of less size and then I remembered that we jumped to node 8 mainly to be able to read async/await easily if something breaks. And long story short, I kept it like this for potentially read code in lambda functions when needed, but i'll change to optimize and we can un-optimize when needed.